assets/js/app/Panel.js

```diff
@@ -527,15 +686,22 @@ LocusZoom.Panel.prototype.removeDataLayer = function(id){
     return this;
 };
 
-// Clear all selections on all data layers
+/**
+ * Clear all selections on all data layers
+ * @returns {LocusZoom.Panel}
+ */
 LocusZoom.Panel.prototype.clearSelections = function(){
     this.data_layer_ids_by_z_index.forEach(function(id){
         this.data_layers[id].setAllElementStatus("selected", false);
     }.bind(this));
     return this;
 };
 
-// Re-Map a panel to new positions according to the parent plot's state
+/**
+ * When the parent plot changes state, adjust the panel accordingly. For example, this may include fetching new data
+ *   from the API as gene position changes
```

Comment:
Again position isn't the best term and "gene position" can be more misleading as not all LZ configurations need to include a genes data layer. Better to use region.

> For example, this may include fetching new data from the API as region is changed.